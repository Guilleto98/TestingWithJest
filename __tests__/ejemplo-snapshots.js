const cliente = {
    nombre: 'Guillermo',
    balance: 505,
    tipo: 'Premium',
}

describe('Testing al cliente', ()=>{
    test('Es Guillermo', ()=>{
        expect(cliente).toMatchSnapshot()
    })
})