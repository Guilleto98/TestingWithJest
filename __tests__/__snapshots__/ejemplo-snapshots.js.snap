// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Guillermo 1`] = `
{
  "balance": 505,
  "nombre": "Guillermo",
  "tipo": "Premium",
}
`;
